src/Components/FieldDisplay.tsx

```diff
@@ -62,7 +62,7 @@ export function NodeFieldDisplay({ node, oldValue, suffix, component, emphasize
   const isTeamBuff = node.info.isTeamBuff
   const fieldFormulaText = node.formula
   let fieldVal = "" as Displayable
-  if (oldValue) {
+  if (oldValue !== undefined) {
```

Comment:
Should we just toss this into the current bug fix release? Unless you are just planning to release this right now

Reply:
I'm moving this to a separate bugbash PR